Example/Backpack/View Controllers/BPKSpacingsViewController.m

```diff
@@ -0,0 +1,64 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKSpacingsViewController.h"
+#import <Backpack/Font.h>
+#import <Backpack/Color.h>
+#import <Backpack/Spacing.h>
+
+@interface BPKSpacingsViewController ()
+@property (strong, nonatomic) IBOutletCollection(UILabel) NSArray *labels;
+@end
+
+@implementation BPKSpacingsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    [self setSpacings];
+}
+
+#pragma mark - Private
+
+- (void)setSpacings {
+    NSArray<NSString *> *spacingKeys = @[
```

Comment:
Use a `NSDictionary` instead

Reply:
I did use that initially but apparently they're unordered, so using the labels iterator to pull out spacings wouldn't be reliable.